org-formation/710-tgw/_tasks.yaml

```diff
@@ -36,7 +36,6 @@ TransitGateway:
       - !Ref BridgeProdAccount
       - !Ref SynapseDevAccount
       - !Ref SynapseProdAccount
-      - !Ref AgoraDevAccount
```

Comment:
Why is Agora being removed from transit gateway? Does this mean that Agora v3 doesn't need a vpc for vpn-only access?

Reply:
The VPC for this agora deployment is being deleted.  Also I believe we initially set this up to allow access to resources in the VPC thru the VPN but now we use SSM sessions therefore it's not really needed anymore.